tests/myproj/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ add_subdirectory(libs)
 
 ## tests
 enable_testing()
-add_test(NAME main COMMAND main)
+add_executable_test(main no_arg)
```

Comment:
The `add_executable_test` actually creates a test named `test.<executable>.<test_name>`. Since users might write multiple tests for an executable, `no_arg` here is just a name saying this test has no arg.